ocsgw/src/main/java/org/ostelco/ocsgw/datasource/protobuf/PubSubDataSource.kt

```diff
@@ -51,29 +51,11 @@ class PubSubDataSource(
             pubSubChannelProvider = FixedTransportChannelProvider.create(GrpcTransportChannel.create(channel))
         }
 
-        // init publisher
-        logger.info("Setting up Publisher for pubsub Topic: {}", ccrTopicId)
         publisher = setupPublisherToTopic(projectId, ccrTopicId)
+        initCcrKeepAlive()
```

Comment:
In existing code, the receivers were set before keep-alive.  Does that matter?

Reply:
Not sure, will fix it